articles/ctod.dd

```diff
@@ -88,8 +88,8 @@ unsigned short     =&gt;        ushort
 wchar_t            =&gt;        wchar
```

Comment:
@ljmf00 Can you also fix this:

```suggestion
wchar_t            =&gt;        core.stdc.stddef.wchar_t
```

(For reference: https://github.com/dlang/druntime/blob/v2.098.0/src/core/stdc/stddef.d#L27-L36)

Reply:
I didn't notice it, thanks. Fixed